src/components/taskDetails/index.tsx

```diff
@@ -289,6 +300,14 @@ const TaskDetails: FC<Props> = ({ taskID }) => {
                     </section>
                 </div>
             )}
+            {dev && userData && taskDetails && (
```

Comment:
Instead of dev, something more descriptive can be better, like `isTaskRequestEnabled`